templates/embedded/form-data.json

```
@@ -0,0 +1,289 @@
+            },
+            {
+              "type": "radio",
+              "label": "100-1000 units",
+              "value": "100-1000 units",
+              "id": "100-1000-units"
+            },
+            {
+              "type": "radio",
+              "label": "1000-5,000 units",
+              "value": "1000-5,000 units",
+              "id": "1000-5000-units"
+            },
+            {
+              "type": "radio",
+              "label": "5,000-25,000 units",
+              "value": "5,000-25,000 units",
+              "id": "5000-25000-units"
+            },
+            {
+              "type": "radio",
+              "label": "25,000-50,000 units",
+              "value": "25,000-50,000 units",
+              "id": "25000-50000-units"
+            },
+            {
+              "type": "radio",
+              "label": "50,000-100,000 units",
+              "value": "50,000-100,000 units",
+              "id": "50000-100000-units"
+            },
+            {
+              "type": "radio",
+              "label": "100,000+ units",
+              "value": "100,000+ units",
+              "id": "100000-units"
+            }
+          ]
+        },
+        {
+          "title": "What is your selected hardware?",
+          "id": "selected-hardware",
+          "inputType": "radio",
+          "isRequired": true,
+          "fields": [
+            {
+              "type": "radio",
+              "label": "Intel",
+              "value": "Intel",
+              "id": "intel"
+            },
+            {
+              "type": "radio",
+              "label": "Arm",
+              "value": "Arm",
+              "id": "arm"
+            },
+            {
+              "type": "radio",
+              "label": "Raspberry Pi/CM",
+              "value": "Raspberry Pi/CM",
+              "id": "raspberry-pi"
+            },
+            {
+              "type": "radio",
+              "label": "NXP",
+              "value": "NXP",
+              "id": "nxp"
+            },
+            {
+              "type": "radio",
+              "label": "Nvidia",
+              "value": "Nvidia",
+              "id": "nvidia"
+            },
+            {
+              "type": "radio",
+              "label": "Qualcomm",
+              "value": "Qualcomm",
+              "id": "qualcomm"
+            },
+            {
+              "type": "radio",
+              "label": "Texas Instruments",
+              "value": "Texas Instruments",
+              "id": "texas-instruments"
+            },
+            {
+              "type": "radio",
+              "label": "Rockchip",
+              "value": "Rockchip",
+              "id": "rockchip"
+            },
+            {
+              "type": "radio",
+              "label": "Not sure",
+              "value": "Not sure",
+              "id": "not-sure"
+            },
+            {
+              "type": "radio",
+              "id": "other",
+              "label": "Other",
```

```suggestion
              "label": "Other - please describe",
```
As on the [live form](https://ubuntu.com/embedded#get-in-touch)